docs/src/main/tut/typeclasses/contravariantmonoidal.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: docs
+title: "Contravariant Monoidal"
+section: "typeclasses"
+source: "core/src/main/scala/cats/ContravariantMonoidal.scala"
+scaladoc: "#cats.ContravariantMonoidal"
+---
+# Contravariant Monoidal
+
+The `ContravariantMonoidal` type class is for [`Contravariant`](contravariant.html) functors that can define a
+`product` function and a `unit` function.
+
+```tut:reset
```

Comment:
Sorry I didn't make it clear. I was suggesting using a `tut:silent` here and `tut:reset:silent` for the next block at line 41.

Reply:
Ah, I see, I was just starting to debug this. Ok, cool, sorry about this, I'm not used to testing against multiple versions of the compiler. I'll fix shortly.

Reply:
Ok, checked it against 2.10.6, 2.11.11, and 2.12.3 and the docs all compile with that recommendation. So we should be GTG.